test/pstl_offload/memory/interop_allocs_system.cpp

```diff
@@ -0,0 +1,25 @@
+// -*- C++ -*-
```

Comment:
What if we just do a symmetric translation unit with USM that has a function like `perform_usm_allocations` and the main what just call `perform_usm_allocations` and `perform_system_allocations` and check the results?

Reply:
Done.